### ISSUE TITLE
get() on simple structures fails

### ISSUE THREAD

**Issue by Tenzer** [thisbejim/Pyrebase#11]
If you only want to get a single item/key in Firebase, which contains one of the primitives Firebase supports (string, number or boolean), `get()` will fail with something like this:

      File "path/to/virtualenv/lib/python3.4/site-packages/pyrebase/pyrebase.py", line 94, in get
        if not isinstance(list(request_dict.values())[0], dict):
    AttributeError: 'str' object has no attribute 'values'

This is with Pyrebase 0.3.1 and with the key I am trying to get being a string. In my [original pull request](https://github.com/thisbejim/Pyrebase/pull/4) for the previous `one()` method, I added a check for if we got a dictionary or something else back, before the id attribute was attempted to be set on the data:

```python
if isinstance(request_json, dict):
    request_json["id"] = item_id
```

In the current version we have a check which is supposed to check the same, as far as I can see, in the interest of returning the raw primitive, if it's not a dictionary, which looks like this:

```python
if not isinstance(list(request_dict.values())[0], dict):
    return request_dict
```

As is evident in the stacktrace, the `values()` method fails on a string, and would in fact fail on anything but a dictionary. I wonder what the reason was behind structuring the check like that. Wouldn't it be better with the simpler method shown on top here?

**Comment by thisbejim** [comment]
Ah yes I see what's happening here. So the new version is checking to see if only one dictionary level is returned, as in: 
```python
{
  "name": "henry"
}
```
rather than 
```python
{
  "-K2QRiOt8oQ5vBYS5mI1":  {
    "name": "henry"
  }, 
  "-K2QfF5GZ2psbso7lEMj": {
    "name": "bob"
  }
}
```
so we don't have to unpack it from a list. Then the first version is checking if we are only returning one primitive e.g. ```"bob"```, so really they should be combined into something like:
```python
# check if dict
if isinstance(request_object.json(), dict):
    # if one level dict return to avoid unpacking list
    if not isinstance(list(request_dict.values())[0], dict):
        return request_dict
else:
    # return primitive
    return request_object.json()
```
Actually that's not quite right, because the request_dict.values() section only checks if the first value is a dict while it is certainly possible the second, third or fourth value could be a dict (as seen in #12). Hmmm will have to think about this a bit.

**Comment by Tenzer** [comment]
Wouldn't this work?
```python
# check if dict
if isinstance(request_object.json(), dict):
    # if one level dict return to avoid unpacking list
    if len(request_object.json()) == 1:
        return request_object.json().popitem()[1]
else:
    # return primitive
    return request_object.json()
```

**Comment by thisbejim** [comment]
Not quite, the one level deep dict would look something like:
```python 
{
  "name": "bob"
  "age": 54
}
```
We just need to return exactly that if none of the keys in the dict contain dict values, e.g don't return:
```python 
{
  "name": "bob"
  "dict": {
    "age": 54
  }
}
```
Actually, we don't even need to do any of that. This is ultimately an attempt to check if the query is returning a bunch of sorted children or just a single one level dict - so you could just keep the primitive check and then check if ```buildQuery``` is being used and if not just return it.

**Comment by thisbejim** [comment]
Should be solved with f7dadcf1ab1559c86bcba37c97d3037de0c0d495
